pkg/resource/training_job/testdata/training_job/cr/traij_cmd_invalid_before_create.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: FeatureGroup
```

Comment:
mismatch Kind, again how are tests passing?

also, file name inconsistent. problem with short forms but ok. call it trainj

Reply:
++ it is concerning that tests are passing

Reply:
It is possible we should validate the desired_state / input yaml file by using the testutil/util.go YamlEqual function.
Suggestion is to do this in a separate PR.